webpack/webpack.config.dev.js

```diff
@@ -86,7 +86,6 @@ export default {
   module: {
     loaders: [
       {test: /\.json$/, loader: 'json-loader'},
-      {test: /\.txt$/, loader: 'raw-loader'},
```

Comment:
We weren't using this, and it wasn't working (I'm not sure why). So, I removed it.